javascript-timers/interval.js

```diff
@@ -0,0 +1,15 @@
+var $h1 = document.querySelector('h1');
+var i = 4;
+
+var interval;
+
+function majorTom() {
+  if (i > 1) {
+    $h1.textContent = (--i).toString();
```

Comment:
There's no need for `toString` here, the DOM API will do that for you when setting the attribute value 🤓